base/ltclass.dtx

```diff
@@ -1116,27 +1116,28 @@
 %    A few more conditionals for convenience
 % \changes{v1.5k}{2024/04/10}{Provide T and F conditionals not just TF
 %    (gh/1262)}
+% \changes{v1.5m}{2025/05/10}{avoid errors for par in code argument of T versions (gh/1733)}
 %    \begin{macrocode}
 %</2ekernel>
 %<*2ekernel|latexrelease>
 %<latexrelease>\IncludeInRelease{2024/06/01}%
 %<latexrelease>                 {\IfPackageLoadedT}{More conditionals}%
-\def\IfPackageLoadedT #1#2{\IfPackageLoadedTF{#1}{#2}{}}
+\def\IfPackageLoadedT   #1{\IfPackageLoadedTF{#1}\@firstofone\@gobble}
 \def\IfPackageLoadedF   #1{\IfPackageLoadedTF{#1}{}}
-\def\IfClassLoadedT   #1#2{\IfClassLoadedTF{#1}{#2}{}}
+\def\IfClassLoadedT     #1{\IfClassLoadedTF{#1}\@firstofone\@gobble}
 \def\IfClassLoadedF     #1{\IfClassLoadedTF{#1}{}}
-\def\IfPackageAtLeastT#1#2#3{\IfPackageAtLeastTF{#1}{#2}{#3}{}}
-\def\IfPackageAtLeastF  #1#2{\IfPackageAtLeastTF{#1}{#2}{}}
-\def\IfClassAtLeastT  #1#2#3{\IfClassAtLeastTF{#1}{#2}{#3}{}}
+\def\IfPackageAtLeastT#1#2{\IfPackageAtLeastTF{#1}{#2}\@firstofone\@gobble}
+\def\IfPackageAtLeastF#1#2{\IfPackageAtLeastTF{#1}{#2}{}}
```

Comment:
Not wrong, but is it "best practice" not to add space before such #1... ?

Reply:
I don't see the practice is good or bad, since it makes no difference. I was just copying the style already used in this block where the code for each of the TF, T and F variants was written so the replacement text aligns even though they have different numbers of arguments. 

Normally (outside L3 code) I would never add a space before `#1` but as I say here the existing style had them to line things up.